languages/en.ini

```diff
@@ -652,6 +652,7 @@ hold_required_by_optional = "No longer required after (optional)"
 hold_start_date = "Start date"
 hold_start_date_invalid = "Please enter a valid start date"
 hold_success = "Your request was successful"
+holding_no_items_availability_message = "See full record"
```

Comment:
I forgot that we actually have a separate namespace for these types of messages -- I'd suggest moving this into languages/HoldingStatus/en.ini and then adjusting the string in Folio.php to be "HoldingStatus::holding_no_items_availability_message".

Also note that you can run `php $VUFIND_HOME/public/index language normalize $VUFIND_HOME/languages` to fix the normalization error that's causing a test failure right now.

Reply:
Oh, wow! I didn't even know those sub-directories were there!

Okay, let's see if the tests pass with this change. They should...